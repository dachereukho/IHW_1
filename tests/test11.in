10
69
31
-67
-80
55
78
-2
23
-74
0