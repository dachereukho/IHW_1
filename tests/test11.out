31 -80 78 23 0 69 -67 55 -2 -74